docs/docs/guides/quick-start/optimize-workflows.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Optimize workflows
+description: Learn how to optimize workflows with Tuist.
+---
+
+# Optimize workflows
+
+Because Tuist knows about your project through your description and the insights it collects, it can optimize your workflows to make them more efficient. Let's see some examples.
```

Comment:
```suggestion
Because Tuist knows your project through your description and the insights it collects, it can optimize your workflows to make them more efficient. Let's see some examples.
```